### PR TITLE
Update ebl_domain.yaml

### PR DESCRIPTION
**Pull request by nissimsan**
Transport document description updated with desc from DCSA's glossary.

### CODE REVIEW

File: domain/ebl_domain.yaml [hunk 1]
```diff
@@ -529,10 +529,6 @@ components:
           properties:
             declaredValue:
               $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/declaredValue'
-        - type: object
-          properties:
-            numberOfRiderPages:
-              $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/1.0.0#/components/schemas/numberOfRiderPages'
```
Comment (HenrikHL) [inline domain/ebl_domain.yaml:535]: Is this a mistake?

Comment (nissimsan) [reply domain/ebl_domain.yaml:535]: Oh, yea, forgot to branch. Not a mistake otherwise, but something completely different

Comment (HenrikHL) [reply domain/ebl_domain.yaml:535]: Will you make a separate PR for removing the numberOfRiderPages? I think that parts needs more discussion ;-) I do agree with your suggestion to update the TransportDocument description

Comment (nissimsan) [reply domain/ebl_domain.yaml]: Done - sorry for the slobbyness :)

**Review by HenrikHL — APPROVED** [review]

**Pull request merged**